Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -551,22 +551,33 @@ export type Props = $ReadOnly<{|
  */
 const ScrollView = createReactClass({
   displayName: 'ScrollView',
-  mixins: [ScrollResponder.Mixin],
-
+  _scrollResponder: {},
   _scrollAnimatedValue: (new AnimatedImplementation.Value(
     0,
   ): AnimatedImplementation.Value),
   _scrollAnimatedValueAttachment: (null: ?{detach: () => void}),
   _stickyHeaderRefs: (new Map(): Map<number, ScrollViewStickyHeader>),
   _headerLayoutYs: (new Map(): Map<string, number>),
+  _createScrollResponder: function() {
+    for (const key in ScrollResponder.Mixin) {
+      if (typeof ScrollResponder.Mixin[key] === 'function') {
+        this._scrollResponder[key] = ScrollResponder.Mixin[key].bind(
+          this._scrollResponder,
+        );
+      }
+    }
+    this._scrollResponder.props = this.props;
+    this._scrollResponder.state = ScrollResponder.Mixin.scrollResponderMixinGetInitialState();
+  },
   getInitialState: function() {
+    this._createScrollResponder();
```

Comment:
I'm not too sure it's a good idea to hack `getInitialState` to initialize the scrollResponder. What we really need is a constructor, and we can get that by converting this component to a class, which shouldn't take too much time. I think we should just do that instead. Besides, we're going to have to remove this code eventually when we convert to a class down the line anyways.